node-api/handlers/beacon/header.go

```diff
@@ -70,16 +66,12 @@ func (h *Handler) GetBlockHeaderByID(c handlers.Context) (any, error) {
 	if err != nil {
 		return nil, err
 	}
-	return beacontypes.ValidatorResponse{
-		ExecutionOptimistic: false, // stubbed
-		Finalized:           false, // stubbed
-		Data: &beacontypes.BlockHeaderResponse{
-			Root:      header.GetBodyRoot(),
-			Canonical: true,
-			Header: &beacontypes.SignedBeaconBlockHeader{
-				Message:   beacontypes.BeaconBlockHeaderFromConsensus(header),
-				Signature: "", // TODO: implement
-			},
+	return beacontypes.NewResponse(&beacontypes.BlockHeaderResponse{
```

Comment:
so here we just wrapped the `beacontypes.BlockHeaderResponse` in the `NewResponse` here. No introducing a new TODO here

Reply:
right, the TODO for Signature was already there, so kept it as in.